tox.ini

```diff
@@ -9,8 +9,7 @@ extras =
     test
 deps =
     coverage>=4.2
-    cryptographyMaster: git+https://github.com/pyca/cryptography.git
-    cryptographyMinimum: cryptography==3.3
```

Comment:
Without this line, how does -minimum work?

Reply:
Huh, when I told magit to revert that line I wasn't expecting it to delete it entirely. A bit disturbing that nothing breaks with that missing. Anyway, nice catch, thanks :-) And I reverted it properly now...